datasm/scripts/check_file_integrity.py

```diff
@@ -33,6 +33,7 @@ def check_file(path):
 
     if not out or "NetCDF: HDF error" in err:
         con_message("error", f"Error loading {path}")
+        log_message("info", f"DEBUG: Error loading {path}")
```

Comment:
```suggestion
        log_message("debug", f"Error loading {path}")
```
Should this be a `debug` level message, rather than `info`?

Reply:
Yes.  The issue is, if I run with --debug, I will get 1000 times as much output, and I only wanted to "peek" at certain values.  I should remove one or the other.

In fact, an "error" should be a log_message("error", "..."), no?

(There should be a "debug_<tag>" ability, so that only messages of the matching tag get printed...)

Reply:
I see what you're saying. I don't think you want "ERROR" because you're using this line for debugging purposes.

The default logger module level is "WARNING". If you set this line as "DEBUG" ("debug"), it won't appear (unless the default level is set to "DEBUG" in datasm). Refer to the table below for the level hierarchy.

https://docs.python.org/3/howto/logging.html:
<div class="responsive-table__container">

Level | When it’s used
-- | --
DEBUG | Detailed information, typically of interest only when diagnosing problems.
INFO | Confirmation that things are working as expected.
WARNING | An indication that something unexpected happened, or indicative of some problem in the near future (e.g. ‘disk space low’). The software is still working as expected.
ERROR | Due to a more serious problem, the software has not been able to perform some function.
CRITICAL | A serious error, indicating that the program itself may be unable to continue running.

</div>
<p>The default level is <code class="docutils literal notranslate"><span class="pre">WARNING</span></code>, which means that only events of this level
and above will be tracked, unless the logging package is configured to do
otherwise.</p>

Reply:
So if the default level is "WARNING", then "WARNING", "ERROR", and "CRITICAL" messages will all show.